http/server.lua

```diff
@@ -1295,11 +1297,72 @@ local function url_for_httpd(httpd, name, args, query)
     end
 end
 
+local function enable_tls(host, port, ssl_opts)
+    local ok, ctx = pcall(sslsocket.ctx, ffi.C.TLS_server_method())
```

Comment:
Is it possible? I wouldn't like to spread `ffi` around the code.
```suggestion
    local ok, ctx = pcall(sslsocket.ctx, sslsocket.tls_server_method())
```